### PR TITLE
Documentation update: external chaincode and index for db

### PR DESCRIPTION
**Pull request by andreevym**
- Documentation update
add information about metadata dir with index in external chaincode

### CODE REVIEW

File: docs/source/cc_service.md [hunk 1]
```diff
@@ -28,6 +28,13 @@ The chaincode package should be used to provide two pieces of information to the
 * provide chaincode endpoint information in a `connection.json` file placed in the release directory. The `bin/run` section describes the `connection.json` file
 
 There is plenty of flexibility to gathering the above information. The sample scripts in the [External builder and launcher sample scripts](#external-builder-and-launcher-sample-scripts) illustrate a simple approach to providing the information.
+As an example of flexibility, consider packaging couchdb index files ( see https://hyperledger-fabric.readthedocs.io/en/release-1.4/couchdb_tutorial.html#add-the-index-to-your-chaincode-folder
```
Comment (nikhil550) [inline docs/source/cc_service.md]: Use a real link here. eg: see [Add the index to your chaincode folder](couchdb_tutorial.html#add-the-index-to-your-chaincode-folder)

**Review by nikhil550 — DISMISSED** [review]

File: docs/source/cc_service.md (hunk 1 above)
Comment (andreevym) [reply docs/source/cc_service.md]: thanks. done

File: docs/source/cc_service.md [hunk 2]
```diff
@@ -28,8 +28,7 @@ The chaincode package should be used to provide two pieces of information to the
 * provide chaincode endpoint information in a `connection.json` file placed in the release directory. The `bin/run` section describes the `connection.json` file
 
 There is plenty of flexibility to gathering the above information. The sample scripts in the [External builder and launcher sample scripts](#external-builder-and-launcher-sample-scripts) illustrate a simple approach to providing the information.
-As an example of flexibility, consider packaging couchdb index files ( see https://hyperledger-fabric.readthedocs.io/en/release-1.4/couchdb_tutorial.html#add-the-index-to-your-chaincode-folder
-) (`/metadata/statedb/couchdb/indexes/`). Sample scripts below describe an approach to packaging the files into code.tar.gz.
+As an example of flexibility, consider packaging couchdb index files (see [Add the index to your chaincode folder](https://hyperledger-fabric.readthedocs.io/en/release-1.4/couchdb_tutorial.html#add-the-index-to-your-chaincode-folder)) (`/metadata/statedb/couchdb/indexes/`). Sample scripts below describe an approach to packaging the files into code.tar.gz.
```
Comment (muralisrini) [inline docs/source/cc_service.md]: ` (/metadata/statedb/couchdb/indexes/)` seems out of place ? Should be removed ?

Comment (andreevym) [reply docs/source/cc_service.md]: sure, okay

**Review by denyeart — APPROVED** [review]

**Pull request merged**